CHANGES.md

```diff
@@ -24,6 +24,56 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.29.0
+
+- educational materials
+- Doc update for `responsibleai_vision` by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2164
```

Comment:
it looks like you are missing an indent here - 30 to 35 need to be indented under "educational materials"

Reply:
actually it looks like all of these lines are indented incorrectly - they need to be under the group header, otherwise they appear at the same level

Reply:
thanks! updated!